utapiServer.js

```diff
@@ -1,4 +0,0 @@
-'use strict'; // eslint-disable-line strict
```

Comment:
I agree that this is not ideal and we need proper directory structure ( which I am passively working on). I wanted to clean up unnecessary files from the root folder and I think that this is a good compromise interim.